var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -70,9 +70,15 @@ def __build_mods_coreneuron(self, mods_location, link_flag, include_flag):
         spec = self.spec
         assert os.path.isdir(mods_location), mods_location
         include_flag += ' -I%s' % (spec['coreneuron'].prefix.include)
-        which('nrnivmodl-core')(
-            '-i', include_flag, '-l', link_flag, '-n', self.mech_name,
-            '-v', str(spec.version), '-c', mods_location)
+        if (str(spec['coreneuron'].version) == 'develop') or (str(spec['coreneuron'].version) ==
```

Comment:
this should be `spec.satisfies('coeneuron@develop')`

Reply:
@jorblancoa suggested to add the check on master version. Are you saying I should revert back?

Reply:
hmm..I see the master version referenced in package recipe (which should be removed soon). You can take care of comparing version by using `satisfies` and then we can merge.